resources/language/resource.language.nl_nl/strings.po

```diff
@@ -93,12 +93,12 @@ msgid "Rate on Netflix"
 msgstr "Op Netflix beoordelen"
 
 msgctxt "#30020"
-msgid "Remove from 'My list'"
```

Comment:
Has the msgid changed?

Reply:
no the id is not changed, original id must be keep